packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -199,6 +210,12 @@ class DefaultWidgetsLocalizations implements WidgetsLocalizations {
   static Future<WidgetsLocalizations> load(Locale locale) {
```

Comment:
i think we have to move the rtl logic here into the new package too, otherwise in the `ar` locale, it'll be english but flipped.

Reply:
Good point, we don't want a non-i18n WidgetsApp app to display RTL. That's going to complicate configuring real i18n apps a little.

Reply:
I've moved the widgets localizations into the flutter_localizations package (see GlobalWidgetsLocalizations). DefaultWidgetsLocalizations.textDirection is wired to LTR.